tests/color_glyph_test.py

```diff
@@ -464,15 +464,15 @@ def _round_coords(paint, prec=5):
                     source=PaintColrLayers(
                         layers=(
                             PaintGlyph(
```

Comment:
It's good to have tests to confirm you reliable emit things backwards but perhaps even better to use the correct order

Reply:
LOL +1